.github/workflows/check-ca-certs.yaml

```
@@ -7,6 +7,13 @@ on:
   schedule:
     # Runs every morning at 2:00 AM UTC
     - cron: "0 2 * * *"
+  pull_request:
+    branches: [main]
+    # milestoned is added here as a workaround for release-please not triggering PR workflows (PRs should be added to a milestone to trigger the workflow).
+    types: [milestoned, opened, reopened, edited, synchronize]
+    paths:
+      - "certs/**"
+      - "src/pepr/uds-operator-config/templates/uds-ca-certs.yaml"
```

Should also include the workflow?

```suggestion
    paths:
      - "certs/**"
      - "src/pepr/uds-operator-config/templates/uds-ca-certs.yaml"
      - ".github/workflows/check-ca-certs.yaml"
```

---

yea good call. added.